test/message/test_runner_only_tests.js

```diff
@@ -46,3 +46,35 @@ test('only = true, with subtests', { only: true }, async (t) => {
   await t.test('skipped subtest 3', { only: false });
   await t.test('skipped subtest 4', { skip: true });
 });
+
+describe.only('describe only = true, with subtests', () => {
+  it('`it` subtest 1 should run', () => {});
+
+  it('`it` subtest 2 should run', async () => {});
+});
+
+describe.only('desribe only = true, with a mixture of subtests', () => {
+  it.only('`it` subtest 1', () => {});
+
+  it.only('`it` async subtest 1', async () => {});
+
+  it('`it` subtest 2 only=false', { only: false }, () => {
+    throw new Error('This should not run');
+  });
```

Comment:
Can we test that mixing both syntax still works?
```suggestion
  it('`it` subtest 2 only=true', { only: true });

  it('`it` subtest 2 only=false', { only: false }, () => {
    throw new Error('This should not run');
  });
```

Reply:
Sure, done here https://github.com/nodejs/node/commit/c7c895ae6c07e6ed6da3c28effdb0c40e48e0314.